### PR TITLE
[p5js - HI] Translation of shape and color pages to hindi

### PR DESCRIPTION
**Pull request by jaiakash**
Hi, i have completed the shapes and color pages translation in hindi. @TanviKumar @Divyansh013 can you please review this PR and let me know the changes if any.

Thanks !

### CODE REVIEW

**Review by jaiakash — COMMENTED** [review]
Self-review one pass

File: src/content/reference/hi/p5/brightness.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: brightness
+module: Color
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/brightness.mdx:3]: Translate module and submodule name

File: src/content/reference/hi/p5/erase.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+title: erase
+module: Color
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/erase.mdx:3]: Here also translate module and submodule

File: src/content/reference/hi/p5/green.mdx [hunk 3]
```diff
@@ -0,0 +1,147 @@
+---
+title: green
+module: रंग
+submodule: बनाना और पढ़ना
+file: src/color/creating_reading.js
+description: >
+  <p>किसी रंग का हरा मान प्राप्त होता है।</p>
+
+  <p><code>green()</code> a से हरा मान निकालता है
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/green.mdx:9]: 'a' here is unnecessary right?

File: src/content/reference/hi/p5/noSmooth.mdx [hunk 4]
```diff
@@ -0,0 +1,81 @@
+---
+title: noSmooth
+module: आकार
+submodule: गुण
+file: src/core/shape/attributes.js
+description: >
+  <p>दांतेदार (उर्फ) किनारों के साथ कुछ विशेषताएं बनाता है।</p>
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/noSmooth.mdx:7]: sounds bit complex,  can you suggest some alternatives?

File: src/content/reference/hi/p5/rectMode.mdx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+      describe('A small gray square drawn at the center of a white square.');
+    }
+    </code>
+    </div>
+class: p5
+params:
+  - name: mode
+    description: |
+      <p>या तो कोना, कोने, केंद्र, या त्रिज्या</p>
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/rectMode.mdx:106]: Check this once.

File: src/content/reference/hi/p5/strokeCap.mdx [hunk 6]
```diff
@@ -0,0 +1,58 @@
+---
+title: strokeCap
+module: आकार
+submodule: गुण
+file: src/core/shape/attributes.js
+description: >
+  <p>पंक्तियों के सिरों को प्रस्तुत करने के लिए शैली सेट करता है।</p>
+
+  <p>पंक्ति के अंत के कैप या तो गोलाकार (<कोड>गोल</कोड>), वर्गाकार (<कोड>वर्ग</कोड>), या विस्तारित (<कोड>प्रोजेक्ट</कोड>) होते हैं। डिफ़ॉल्ट सीमा <कोड>राउंड</कोड> है।</p>
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/strokeCap.mdx:9]: don't translate "code"

**Review by Divyansh013 — CHANGES_REQUESTED** [review]
Have left some reviews, please have a look. Thanks!

File: src/content/reference/hi/p5/brightness.mdx (hunk 1 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/brightness.mdx]: Done

File: src/content/reference/hi/p5/green.mdx (hunk 3 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/green.mdx]: Yeah, i have removed it now.

File: src/content/reference/hi/p5/erase.mdx (hunk 2 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/erase.mdx]: Done

File: src/content/reference/hi/p5/strokeCap.mdx (hunk 6 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/strokeCap.mdx]: My bad, changed it.

File: src/content/reference/hi/p5/rectMode.mdx (hunk 5 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/rectMode.mdx]: I have used simpler language for it.

File: src/content/reference/hi/p5/noSmooth.mdx (hunk 4 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/noSmooth.mdx]: Yeah sure, made it simpler.

**Review by Divyansh013 — APPROVED** [review]
Looks good. Thanks @jaiakash

**Review by TanviKumar — APPROVED** [review]

**Comment by Divyansh013** [comment]
Hi @limzykenneth @Qianqianye we can merge this PR!

**Comment by Qianqianye** [comment]
Thanks @jaiakash @TanviKumar @Divyansh013!

**Pull request merged**